dvc/analytics.py

```diff
@@ -65,7 +70,7 @@ def send(path):
     """
     import requests
 
-    url = "https://analytics.dvc.org"
+    url = os.environ.get(DVC_ANALYTICS_HOST, "https://analytics.dvc.org")
```

Comment:
Maybe handle as an optional debug flag instead of env var? So it would need to be explicitly passed to the command